packages/sdk/src/modules/rbac/index.ts

```diff
@@ -517,35 +529,28 @@ export class RBAC extends Base {
 
   /**
    * Fetch all roles.
-   * @param ownerAddress - The ownerAddress is public address of user or owner who created a roles.
+   * @param options - The ownerAddress is public address of user or owner who created a roles.
    * @returns A promise that resolves when the role is fetched.
    */
 
-  public async fetchRoles(ownerAddress: Address): Promise<FetchRoles[]> {
+  public async fetchRoles(options: FetchRoles): Promise<ResponseFetchGroup[]> {
```

Comment:
Fetch all (for Roels, groups, and permissions) now expect an object, rather than just an address. This makes it so you can use {} syntax when calling the functions.